onebusaway-android/src/main/java/com/joulespersecond/seattlebusbot/ArrivalsListFragment.java

```diff
@@ -891,10 +892,26 @@ private void refreshSituations(List<ObaSituation> situations) {
         mSituationAlerts = new ArrayList<SituationAlert>();
 
         for (ObaSituation situation : situations) {
-            SituationAlert alert = new SituationAlert(situation);
-            mSituationAlerts.add(alert);
+            if(isActiveWindowForSituation(situation)) {
+                SituationAlert alert = new SituationAlert(situation);
+                mSituationAlerts.add(alert);
+            }
         }
         mAlertList.addAll(mSituationAlerts);
     }
 
+    private boolean isActiveWindowForSituation(ObaSituation situation) {
+        boolean isActiveWindowForSituation = situation.getActiveWindows().length == 0;
```

Comment:
Sorry, just kidding.  I had a bug in the test data.  I'll blame it on lack of sleep ;).  This implementation works as intended, although I'm still going to push a new PR that includes the tests and refactoring of this to allow testing.  Thanks for putting this together @softrunit!